resources/schema.py

```diff
@@ -632,6 +646,9 @@ def resolve_piers(self, info, **kwargs):
             "harbor__municipality__translations",
         ).select_related("harbor", "harbor__availability_level", "harbor__municipality")
 
+    def resolve_harbor_by_servicemap_id(self, info, **kwargs):
+        return Harbor.objects.get(servicemap_id=kwargs.pop("servicemap_id", None))
```

Comment:
I would probably do `filter().first()` to ensure that no error is raised when wrong `servicemap_id` passed